validator/node/node.go

```diff
@@ -59,10 +60,11 @@ func NewValidatorClient(ctx *cli.Context) (*ValidatorClient, error) {
 	}
 
 	var pubKey []byte
+	var err error
 
 	inputKey := ctx.GlobalString(types.PubKeyFlag.Name)
 	if inputKey == "" {
-		var err error
+
 		pubKey, err = GeneratePubKey()
```

Comment:
What kind of public key is this generating? How is the private key getting handled here?

Reply:
Ah , I think this was part of the demo , so if no pubkey is inputted the node would generate a random one. I think with private key management being functional , there will be no need for this. Will remove it